index.html

```diff
@@ -16,6 +16,14 @@
     border-collapse: collapse;
   }
 
+  #stages {
+    table-layout: fixed;
+    width: 100%;
+  }
+  #stages th:first-child {
```

Comment:
there's only one table, so presumably this doesn't need the ID at all, and could be combined with the existing `table { }` declaration?

Reply:
I guess but that kinda leaves a hazard if we ever add another table. I don't see a reason not to just say the thing we mean.

Reply:
then should the other one be changed to reflect the ID?

Reply:
It is plausible that the author intended that to apply to all tables. I think it's a common "default override".